automator/automator.py

```diff
@@ -125,10 +120,11 @@ def parse_msg(self, msg):
             return
 
         subarray_state, subarray_name = msg_components
+        log.info('subarray {} is now in state: {}'.format(subarray_name, subarray_state))
```

Comment:
just a minor detail, but now that you're using f-strings in some places, you might as well use them in new places. the "format" way of formatting a string is the old and outdated one now afaict

Reply:
Thanks, have changed them for consistency